test/extended/setup.sh

```diff
@@ -155,11 +155,11 @@ readonly EXCLUDED_TESTS=(
   "RollingUpdateDeployment should delete old pods and create new ones"
   "RecreateDeployment should delete old pods and create new ones"
 
-  "\[Feature:PodAffinity\]" # feature is disabled
-  Ingress                 # Not enabled yet
-  "should proxy to cadvisor" # we don't expose cAdvisor port directly for security reasons
-  "Cinder"                # requires an OpenStack cluster
-  "should support r/w"    # hostPath: This test  expects that host's tmp dir is WRITABLE by a container.  That isn't something we need to gaurantee for openshift.
+  "\[Feature:Federation\]"   # Not enabled yet
```

Comment:
Federation isn't started by the master, so it's more being tracked by the
card to enable federation (wherever that is).

On Thu, Jul 7, 2016 at 3:38 PM, David Eads <notifications@github.com> wrote:

> In test/extended/setup.sh
> <https://github.com/openshift/origin/pull/9578#discussion_r69972589>:
>
> > @@ -155,11 +155,11 @@ readonly EXCLUDED_TESTS=(
> >    "RollingUpdateDeployment should delete old pods and create new ones"
> >    "RecreateDeployment should delete old pods and create new ones"
> >
> > -  "\[Feature:PodAffinity\]" # feature is disabled
> > -  Ingress                 # Not enabled yet
> > -  "should proxy to cadvisor" # we don't expose cAdvisor port directly for security reasons
> > -  "Cinder"                # requires an OpenStack cluster
> > -  "should support r/w"    # hostPath: This test  expects that host's tmp dir is WRITABLE by a container.  That isn't something we need to gaurantee for openshift.
> > +  "\[Feature:Federation\]"   # Not enabled yet
>
> If you want someone to look at this, you should create an issue.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/97e9d1c9e8d57b370f2e34580e1871f4299117c6..cfedbc1708fc75ac73aba9d0e9e115b906cd8527#r69972589>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p5Dw__LJxgt5Yg3Bk3DJIRGF-ZXhks5qTVXMgaJpZM4I-xTE>
> .
>